### PR TITLE
Cyberbotics admin rights to delete animations

### PR DESCRIPTION
**Pull request by ThomasOliverKimble**
This PR gives  admins (users with _*****@cyberbotics.com_ emails) the right to delete any animation/scene.
It also contains a few quick fixes in `uniqueness.php`.
I have added 4 dummy scenes to [testing.webots.cloud](https://testing.webots.cloud/scene) for you to test this on.

### CODE REVIEW

File: httpdocs/js/webots-cloud.js [hunk 1]
```diff
@@ -159,11 +159,12 @@ document.addEventListener('DOMContentLoaded', function() {
           duration = hour + duration;
         }
       }
+      const admin = project.email ? (project.email.endsWith('@cyberbotics.com') ? true : false) : false;
```
Comment (omichel) [inline httpdocs/js/webots-cloud.js]: Simpler like that:
```suggestion
      const admin = project.email ? project.email.endsWith('@cyberbotics.com') : false;
```

File: httpdocs/ajax/animation/delete.php [hunk 2]
```diff
@@ -16,7 +16,12 @@ function error($message) {
 $user = isset($data->user) ? intval($data->user) : 0;
 $animation = intval($data->animation);
 $password = isset($data->password) ? $mysqli->escape_string($data->password) : '';
-$query = "DELETE FROM animation WHERE id=$animation AND (user=0 OR user IN (SELECT id FROM user WHERE id=$user AND password=\"$password\"))";
+$query = "SELECT id FROM user WHERE id=$user AND email LIKE '%@cyberbotics.com'";
+$result = $mysqli->query($query) or error($mysqli->error);
+$admin = $result->fetch_assoc();
+$query = "DELETE FROM animation WHERE id=$animation";
+if (!$admin)
+  $query .= " AND (user=0 OR user IN (SELECT id FROM user WHERE id=$user AND password=\"$password\"))";
```
Comment (omichel) [inline httpdocs/ajax/animation/delete.php]: I believe something like that would be simpler:
```suggestion
  $query = "DELETE FROM animation WHERE id=$animation AND (user=0 OR user IN (SELECT id FROM user WHERE id=$user AND password='$password') OR EXISTS (SELECT * FROM user WHERE id=$user AND password='$password' AND email LIKE '%@cyberbotics.com')";
```

**Review by omichel — CHANGES_REQUESTED** [review]
It works nicely, however I believe we could me some simplifications.

**Comment by ThomasOliverKimble** [comment]
> It works nicely, however I believe we could me some simplifications.

Simplifications have been made, thank you.

**Review by omichel — APPROVED** [review]
Nice.

**Pull request merged**